model/src/main/kotlin/utils/DependencyGraphBuilder.kt

```diff
@@ -387,15 +387,18 @@ private fun Collection<DependencyReference>.toGraph(): Pair<List<DependencyGraph
     val edges = mutableListOf<DependencyGraphEdge>()
```

Comment:
Commit message: `guarateed` -> `guaranteed`

Reply:
Also:

* "creates an edge" -> "created an edge"
* "but hasn't created" -> "but didn't create"
* "Note: the previous" -> "Note: The previous"